README.md

```diff
@@ -232,6 +230,7 @@ async def embed(engine: AsyncEmbeddingEngine):
 
 asyncio.run(embed(array["laion/clap-htsat-unfused"]))
 ```
+*Note:* Audio Files need to have a sample rate of 48kHz.
```

Comment:
Already fixed in https://github.com/michaelfeil/infinity/pull/374 @wirthual #375 

The sampling rate needs to be the same as the model, not generally 48k